happypose_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 3.10)
+project(happypose_msgs)
+
+find_package(ament_cmake REQUIRED)
```

Comment:
For future reference, concerning pure `ament` package like this please consider `ament_cmake_auto`.

Reply:
Due to an additional package `happypose_msgs_py` being created on a build time, I decided to resort to default ament